Modules/_cursesmodule.c

```diff
@@ -4738,22 +4744,24 @@ curses_destructor(PyObject *op)
 PyMODINIT_FUNC
 PyInit__curses(void)
 {
-    PyObject *m, *d, *v, *c_api_object;
-
     /* Initialize object type */
-    if (PyType_Ready(&PyCursesWindow_Type) < 0)
+    if (PyType_Ready(&PyCursesWindow_Type) < 0) {
         return NULL;
+    }
+
+    static void *PyCurses_API[PyCurses_API_pointers];
```

Comment:
There are warnings on the PR, like ‘PyCurses_API’ defined but not used [-Wunused-variable].